orc8r/gateway/c/common/config/includes/MConfigLoader.h

```diff
@@ -12,41 +12,28 @@
  */
 #pragma once
 
-#include <iosfwd>  // for ifstream
-#include <string>  // for string
-namespace google {
-namespace protobuf {
-class Message;
-}
-}  // namespace google
+#include <istream>                    // for istream
+#include <string>                     // for string
+#include <google/protobuf/message.h>  // for Message
 
 namespace magma {
-
 /**
- * MConfigLoader is used to load mconfig files for service configurations
+ * load_service_mconfig_from_file loads an mconfig from the statically defined
+ * files.
+ * @param service_name - name of service to load
+ * @param message - pointer to protobuf message to load file to. Note that
+ *                  this should match the message defined in mconfigs.proto.
+ * @returns true if message was parsed successfully, false if the file reading
+ *          failed, the service configuration is missing, or the message
+ *          passed is not the right type
  */
-class MConfigLoader {
- public:
-  /**
-   * load_service_mconfig loads an mconfig from the statically defined files.
-   * @param service_name - name of service to load
-   * @param message - pointer to protobuf message to load file to. Note that
-   *                  this should match the message defined in mconfigs.proto.
-   * @returns true if message was parsed successfully, false if the file reading
-   *          failed, the service configuration is missing, or the message
-   *          passed is not the right type
-   */
-  bool load_service_mconfig(
-      const std::string& service_name, google::protobuf::Message* message);
-
- private:
-  static constexpr const char* DYNAMIC_MCONFIG_PATH =
-      "/var/opt/magma/configs/gateway.mconfig";
-  static constexpr const char* CONFIG_DIR        = "/etc/magma";
-  static constexpr const char* MCONFIG_FILE_NAME = "gateway.mconfig";
```

Comment:
just curious, where are those paths now?

Reply:
not my PR :) I just replicated it for a demo - might want to comment on Scott's original PR